src/bundle/Resources/config/deprecated_routing_commerce_legacy_commerce.yaml

```diff
@@ -0,0 +1,392 @@
+ibexa.compatibility_layer.content:
+    resource: 'deprecated_routing_experience_legacy_commerce.yaml.yaml'
```

Comment:
`.yaml.yaml`?

Reply:
Fixed in https://github.com/ibexa/compatibility-layer/pull/50/commits/6220fb6229384fbbe48d6aa255e81ae36185b703